components/mention.js

```diff
@@ -5,7 +5,7 @@ import Link from 'next/link'
 
 export const StaticMention = memo(
   ({ user = {}, className = '', size = 24, children, ...props }) => (
-    <Link href={`/${user.username}`}>
+    <Link href={`/${user.username}`} legacyBehavior>
```

Comment:
Can you ELI5 what this does?

Reply:
i upgraded the version of next.js and it won't stop showing warning about that Link component 
see [docs](https://nextjs.org/docs/pages/api-reference/components/link#legacybehavior:~:text=legacyBehavior-,An,to%20upgrade.%20A%20codemod%20is%20available%20to%20automatically%20upgrade%20your%20code.,-Good%20to%20know)